src/FTPServer/Ftp/CommandHandlers/ListCommandHandlerBase.cs

```diff
@@ -62,65 +62,52 @@ protected override string OnProcess(string message)
 
         protected string BuildShortReply(string[] asFiles)
         {
-            string fileList = string.Join("\r\n", asFiles);
-            fileList += "\r\n";
+            var fileList = string.Join(AnsiSequences.NewLine, asFiles);
+            fileList += AnsiSequences.NewLine;
             return fileList;
         }
 
         protected string BuildLongReply(string[] asFiles)
         {
-            string dir = GetPath(string.Empty);
+            var dir = GetPath(string.Empty);
 
-            var stringBuilder = new StringBuilder();
+            var ab = new StringBuilder();
```

Comment:
You renamed this one to `ab` even though it is still `StringBuilder`.  I don't know much about FTP protocol or whether or not you intended to use AnsiBuilder here, but the name-type mismatch caught my eye as misleading.

Reply:
changed it back to sb.

This was a mistake on my part.